src/rpc/chain.ts

```diff
@@ -1017,11 +1019,22 @@ export class ChainRpc {
                 `Expected the second argument of getPendingTransactions to be a non-negative integer but found ${to}`
             );
         }
+        if (futureIncluded !== true && futureIncluded !== false) {
+            throw Error(
+                "Expected the third argument to be boolean variable" +
+                    futureIncluded
```

Comment:
Would you change it to use [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of `+` operator?